how-to/manage-users.rst

```diff
@@ -48,10 +48,10 @@ user by running ``sudo -iu stg-launchpad``.
 
 .. code::
      
-     LPCONFIG=launchpad-admin /srv/launchpad/code/utilities/anoint-team-member <username> <team-name>
+     LPCONFIG=launchpad-scripts /srv/launchpad/code/utilities/anoint-team-member <username> <team-name>
```

Comment:
We would need to update line 41 if we update this.
But instead of updating these, I would just fix line 42 to ``in-model qastaging juju ssh launchpad-admin/leader`` - which also works.

Reply:
When I try to follow the way you suggested, the command `sudo su launchpad` fails as there is no `launchpad` user. I will update line 41 instead and use the launchpad-scripts-way. I confirmed this is working.